### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by mstein-ck**

### CODE REVIEW

**Review by ymwymw — COMMENTED** [review]
Is this backwards compatible?
If not, we should increment the major version.

File: react-cardknox-ifields/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Changelog
+
+## changelog for version 1.0.2
```
Comment (ymwymw) [inline react-cardknox-ifields/CHANGELOG.md:3]: Version?

File: react-cardknox-ifields/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-cardknox-ifields",
-  "version": "1.1.2",
+  "version": "2.1.2",
```
Comment (ymwymw) [inline react-cardknox-ifields/package.json:3]:
```suggestion
  "version": "2.0.0",
```

**Comment by mstein-ck** [comment]
Whoops

**Review by ymwymw — APPROVED** [review]

**Pull request merged**